app/views/task_managements/_pay.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form_tag charges_path(amount: @amount, task_id: @task.id) do %>
+  <article>
+    <% if flash[:error].present? %>
+      <div id="error_explanation">
+        <p><%= flash[:error] %></p>
+      </div>
+    <% end %>
+    <label class="amount">
+      <span>Amount: <%= @amount %></span>
+    </label>
+  </article>
+  <script src="https://checkout.stripe.com/checkout.js" class="stripe-button"
+          data-key="<%= Rails.configuration.stripe[:publishable_key] %>"
```

Comment:
It is saved in my application.yml file. I have set the environmental variables on heroku